### ISSUE TITLE
Approval for custom data pull

### ISSUE THREAD

**Issue by definedfunctions** [amzn/selling-partner-api-docs#199]
I created an Amazon Seller account because I needed to be approved as a developer so I could pull data for a client.

After slogging through the developer registration process, I'm at yet another standstill because until my application is listed in the marketplace, my client is unable to authorize access to their account.

I'm over 8 weeks into the process of being able to pull their Amazon data and have yet to be able to do anything because of so many hurdles.

**Can someone _please_ give me insight into how long it will take for my app to be reviewed for listing in the marketplace? It's been almost two weeks and I've heard nothing despite countless follow-ups.**

Official support channels have provided exactly zero help.

I also can't help but mention how it seems pointless to have to go through the process of being listed in the app marketplace despite having zero intention of using this with anyone other than my client.  
![Screen Shot 2020-12-10 at 8 19 43 AM](https://user-images.githubusercontent.com/74795067/101784297-f4324880-3ac0-11eb-8eee-cdb76a0753d6.png)

**Comment by 468092882** [comment]
Yes, me too

**Comment by fernlop** [comment]
Why not register as developer in the seller account of your customer? They can get API Access for themselves in a few days and self-authorize it. 
Then you don't even need to implement the OAuth process and can just generate a refresh token in the seller central.

**Comment by definedfunctions** [comment]
@fernlop That was my original plan, but saw too many warnings in official documentation stating that wasn't allowed.  It's only been since reading these threads that I've seen people suggesting that approach.

That said, I'm a month and a half into the process of doing things the "right way" so would rather avoid starting over by applying for developer access from their account if at all possible.

I appreciate the suggestion nonetheless.

**Comment by definedfunctions** [comment]
I don't have the time to go into all of the details, but the request to list my application in the marketplace was denied. The reviewer clearly didn't take the time to read my application. He stated I didn't need to be approval because my app wouldn't be used by other sellers.  I stated very clearly in the request that my app would be used by another seller, hence why I was making the request.  I only clarified that I was intending to deploy it for mass distribution.

This is crazy.

I'm following up in the case to try and clarify. Really hoping they didn't click a button that drops me from their radar now that they think they've completed the case.
